### ISSUE TITLE
Publish screengrab on mavenCentral()

### ISSUE THREAD

**Issue by vanniktech** [fastlane/fastlane#12651]
### Feature Request

It would be nice if screengrab would be hosted on mavenCentral. Currenlty it's only in jcenter()

The motivation behind that is to have a stable source as jcenter / bintray have been flaky lately.

**Comment by joshdholtz** [comment]
@vanniktech Thanks for the suggestion! It looks like we can easily sync to maven central from within jcenter but we just need to acquire the maven account and namespace - which looks like its a whole process 😬 I can start looking into this though 💪

**Comment by vanniktech** [comment]
Yes exactly. You need to open a Jira ticket to claim the domain and after that as long as you use the same group id you're fine. If you need any help - let me know.

**Comment by joshdholtz** [comment]
Ticket has been opened - https://issues.sonatype.org/browse/OSSRH-40189

**Comment by vanniktech** [comment]
They always ask you whether you own the domain - https://fastlane.tools/ - or not. You can safely say that you do though.

**Comment by vanniktech** [comment]
Is this already done?

**Comment by joshdholtz** [comment]
@vanniktech Not yet 😬 Gotta fix some things yet in order to sync to Maven and will be trying to get that finished up this week 

```
Missing Signature: '/tools/fastlane/screengrab/1.2.0/screengrab-1.2.0.pom.asc' does not exist for 'screengrab-1.2.0.pom'.
Missing Signature: '/tools/fastlane/screengrab/1.2.0/screengrab-1.2.0-javadoc.jar.asc' does not exist for 'screengrab-1.2.0-javadoc.jar'.
Missing Signature: '/tools/fastlane/screengrab/1.2.0/screengrab-1.2.0-sources.jar.asc' does not exist for 'screengrab-1.2.0-sources.jar'.
Missing Signature: '/tools/fastlane/screengrab/1.2.0/screengrab-1.2.0.aar.asc' does not exist for 'screengrab-1.2.0.aar'.
Invalid POM: /tools/fastlane/screengrab/1.2.0/screengrab-1.2.0.pom: Project name missing, Project description missing,
Project URL missing, License information missing, SCM URL missing,
Developer information missing
```

**Comment by vanniktech** [comment]
Those are things that are missing in the POM file.

**Comment by joshdholtz** [comment]
@vanniktech Not yet 😬 Gotta fix some things yet in order to sync to Maven and will be trying to get that finished up this week 

```
Missing Signature: '/tools/fastlane/screengrab/1.2.0/screengrab-1.2.0.pom.asc' does not exist for 'screengrab-1.2.0.pom'.
Missing Signature: '/tools/fastlane/screengrab/1.2.0/screengrab-1.2.0-javadoc.jar.asc' does not exist for 'screengrab-1.2.0-javadoc.jar'.
Missing Signature: '/tools/fastlane/screengrab/1.2.0/screengrab-1.2.0-sources.jar.asc' does not exist for 'screengrab-1.2.0-sources.jar'.
Missing Signature: '/tools/fastlane/screengrab/1.2.0/screengrab-1.2.0.aar.asc' does not exist for 'screengrab-1.2.0.aar'.
Invalid POM: /tools/fastlane/screengrab/1.2.0/screengrab-1.2.0.pom: Project name missing, Project description missing,
Project URL missing, License information missing, SCM URL missing,
Developer information missing
```

**Comment by vanniktech** [comment]
@joshdholtz friendly ping. Has this gotten any further?

**Comment by vanniktech** [comment]
Still an issue.

**Comment by vanniktech** [comment]
Still an issue.

**Comment by vanniktech** [comment]
Still an issue ...

**Comment by janpio** [comment]
Could you take another look at this @joshdholtz? Probably the version that is published to jcenter is also out of date now as we merged a PR from @vanniktech. (We should probably also document how all this works if it isn't yet)

**Comment by vanniktech** [comment]
Yeah, another release cut would be nice. Once that is done we should move to AndroidX so these #13810 #13622 get fixed automatically.

**Comment by vanniktech** [comment]
I don't think it has been solved.

**Comment by vanniktech** [comment]
Still an issue.

**Comment by janpio** [comment]
Not solved.

**Comment by janpio** [comment]
See above.

**Comment by vanniktech** [comment]
Nope

**Comment by vanniktech** [comment]
Nope.

**Comment by janpio** [comment]
@joshdholtz?

**Comment by vanniktech** [comment]
Nope.

**Comment by vanniktech** [comment]
Nope

**Comment by joshdholtz** [comment]
Yeah, let's try and get this one done this week 😇

**Comment by henningBunk** [comment]
Still needed

**Comment by hcahoon-honey** [comment]
Jcenter is going away May 1, 2021, this is needed now.

**Comment by jraska** [comment]
https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

> **UPDATE 4/27/2021**: We listened to the community and will keep JCenter as a read-only repository indefinitely. Our customers and the community can continue to rely on JCenter as a reliable mirror for Java packages.

**Comment by joshdholtz** [comment]
Hello! Screengrab is now officially on Maven Central 😊 But...

The Falcon dependency is not yet... So using `jcenter()` is still required in your gradle file for now. We will either need to just wait for Falcon to be added or pull it 🤷‍♂️ 

https://github.com/jraska/Falcon/issues/52

**Comment by jraska** [comment]
Falcon is now on [Maven Central](https://repo1.maven.org/maven2/com/jraska/falcon/) starting with [the version `2.2.0`](https://github.com/jraska/Falcon/releases/tag/2.2.0).
```
implementation 'com.jraska:falcon:2.2.0'
```

**Comment by mariolenci-hf** [comment]
@joshdholtz `falcon:2.2.0` is now on mavenCentral but screengrab is still using `2.1.1` which is only in JCenter.
Can the falcon dependency be updated?

**Comment by jraska** [comment]
@mariolenci-hf 
If screengrab uses Falcon as transitive dependency, only by including 
`implementation 'com.jraska:falcon:2.2.0'`
within you Gradle file next to screengrab would override that transitive dependency and you don't have to wait for any screengrab update.

@joshdholtz sorry for any inconvenience of needing to update the artifact, moving the existing 2.1.1 to Maven Central was a pain and since build tolls update was necessary for Falcon to compile - binary would be different - hence the new version.

**Comment by mariolenci-hf** [comment]
@jraska Thank you for the quick answer. That's indeed what I ended up doing and solved the issue.
